### ISSUE TITLE
Config Question: Why is the Proxy Config like it is?

### ISSUE THREAD

**Issue by Merith-TK** [haveachin/infrared#209]
https://github.com/haveachin/infrared/blob/main/pkg/infrared/server.go#L46

why choose this particular mapping? I cant seem to find any uh... good info on how to do entries in this proxy config otherwise I would consider trying it out over mc-router (which I currently am using)

if that is what looked like it would work for you at the time, 
I say you can use the chunk of code in my config handler here for the environment, 
https://github.com/Merith-TK/busybox64.portable/blob/main/config.go#L42

Basically in my code for that project i store environment variables as a `map[string]string` which allows me to define arbitrary keynames, 
which if altered to be used here, could be used to allow for (example assumes docker network configs for readability)
```yml
proxies:
  example.com: "lobby:25565"
  creative.example.com: "creative:25565"
  survival.example.com: "survival:25565"
```
